### PR TITLE
Fix for #1197

### PR DESCRIPTION
**Pull request by trebmuh**
Fix for #1197

### CODE REVIEW

**Comment by mattbk** [comment]
Don't feel like you have to use exactly my words from #1197.

**Comment by trebmuh** [comment]
I'd say that if they make sense, then they are the right ones!

**Comment by Changaco** [comment]
The `receiving` number being displayed isn't "how much donors want to give you", it's "how much you've received for the current week".

**Comment by trebmuh** [comment]
I've tried a fix.
I'm not too sure if "received" is better than "are/is receiving" in this context.
@Changaco : please review.

**Comment by Changaco** [comment]
I think that's still too confusing. The `receiving` number isn't just what the user received this week, it's also what they can expect to receive next week and every week after that. It's the user's stable income.

**Comment by trebmuh** [comment]
@Changaco yes you're right, this is still confusing.

Let's make a example to clarify it.

User "bob" did have last week 10 donors who gave 0.5€ each. Then he earned 5€ on the **last  payday**. Let says here we are Monday. Since last Wednesday (= last payday), user "bob" got a new donor of 0.5€, then on next payday (=next Wednesday), he's expected to get 5.5€.

What is the `receiving` number to be displayed?

**Comment by Changaco** [comment]
@trebmuh €5.50 (if I understood your question correctly).

**Comment by trebmuh** [comment]
Thank you @Changaco .

So the sentence "You receive {0} per week." means "You are expecting to receive {0} for the next payday". For disambiguation, I propose that we should word it like that.

Then, I propose we rephrase the other sentence such as "But you can't receive money until you've connected an account to receive."

How does that sound to you @mattbk @Changaco and others?

File: www/%username/receiving/index.html.spt [hunk 1]
```diff
@@ -24,15 +24,15 @@ teams = website.db.all("""
...
-<p>{{ _("{0} receives {1} per week.", participant.username, receiving) }}</p>
+<p>{{ _("{0} received {1} for the current week.", participant.username, receiving) }}</p>
 % endif
 
 % if participant.is_person
 
 % if not participant.has_payment_account
-    <p>{{ _("You can't receive money until you've configured a payment account.") }}</p>
+    <p>{{ _("You can't receive money until you've connected an account to receive.") }}</p>
     <p><a class="btn btn-primary" href="{{ participant.path('payment') }}">{{ _("Configure payments") }}</a></p>
```
Comment (mattbk) [inline www/%username/receiving/index.html.spt]: I think this should be phrased something like "Configure an account to receive payments".

**Comment by mattbk** [comment]
@trebmuh, your first sentence may be even more confusing because weekly payday is going away--all Stripe and PayPal (+whatever) transactions happen at once.

If we're going to stick with the illusion of weekly support (and I disagree with that idea, see #1205), maybe "You've received donations that average $X for the next Y weeks"?

**Comment by Changaco** [comment]
Closing in favor of #1297.

**Pull request closed without merging**